bindings/matrix-sdk-ffi/src/sliding_sync.rs

```diff
@@ -214,19 +214,25 @@ impl SlidingSyncRoom {
         };
 
         let mut reset_broadcast_rx = self.client.sliding_sync_reset_broadcast_tx.subscribe();
-        let timeline = timeline.to_owned();
+        let timeline_c = timeline.to_owned();
```

Comment:
What does the `_c` mean? 🤔

Reply:
Clone.

Reply:
riiiight, I don't know if this a rust thing but I generally prefer clear names: if it's a clone then it's a `timeline_clone`, not that I understand why we even need to make that distinction

Reply:
Needs to be a distinct name because the original one is used below. I don't love it either, could also solve it with a separate scope. Actually I think I'll do that instead.